.github/workflows/go-lint-golangci.yml

```diff
@@ -33,9 +34,5 @@ jobs:
           CONFIG_FILE: ${{ inputs.golangci-config-file }}
 
       - name: Run golangci-lint
-        uses: golangci/golangci-lint-action@v6
-        # defaults: gosimple govet ineffassign unused
-        with:
-          version: v1.63.4
-          args: ${{ env.golangci_args }}
```

Comment:
Why were these lines removed?

I understand that the "version" line probably need to be upgraded (i have not yet investigated).

But why was the `env.golangci_args` removed? That wrecks configuration. See the previous stanza in this workflow.